README.md

```diff
@@ -15,10 +16,16 @@ Additional description about the project and its features.
 - Javascript
 - Webpacks
 - React
+- Redux
+- API
 
 ## Live Demo
 
-[Live Demo Link](https://gemmen29.github.io/{Repo-Name}/)
+[Live Demo Link](https://keen-murdock-9ac8fc.netlify.app/)
+
+## Video Link
+
+[Video Link](https://www.loom.com/share/8a24b74c37aa43de8e4706f12bf7af51)
 
 ## Getting Started
 
```

Comment:
- [ ] Kindly change the header title _Authors_ to _Author_ on _line 56_ of your ReadMe, as this project is done solely. This will make your ReadMe appear more professional.